rotkehlchen/db/calendar.py

```diff
@@ -31,6 +31,7 @@ class CalendarEntry(NamedTuple):
     counterparty: str | None
     address: BlockchainAddress | None
     blockchain: SupportedBlockchain | None
+    color: str | None
```

Comment:
The type we use for this should be same as the tag one.

Use that please. It's in types.py
```
T_HexColorCode = str
HexColorCode = NewType('HexColorCode', T_HexColorCode)
```